### ISSUE TITLE
unary operator expected

### ISSUE THREAD

**Issue by jangrewe** [silinternational/ecs-deploy#57]
I'm running ` ecs-deploy -t 120 --max-definitions 5 --cluster ${ECS_CLUSTER} --service-name ${SERVICE} --image ${ECR_REPO}:latest` and i receive this error (while it continues, though):
```
Using image name: 1234567890.dkr.ecr.eu-west-1.amazonaws.com/frontend-v2:latest
Current task definition: arn:aws:ecs:eu-west-1:1234567890:task-definition/frontend-v2_blue:280
New task definition: arn:aws:ecs:eu-west-1:1234567890:task-definition/frontend-v2_blue:281
/usr/bin/ecs-deploy: line 325: [: !=: unary operator expected
Service updated successfully, new task definition running.
[...]
```

**Comment by fillup** [comment]
@jangrewe can you try again with code in ```develop``` branch? I just merged in #58 with may have fixed your issue.

**Comment by jangrewe** [comment]
looks fine now, thanks!
